nodestream/project/storage.py

```diff
@@ -9,14 +9,17 @@ class StoreConfiguration:
     name: str
     storage_type: str
     arguments: Dict[str, Any]
-    hmac_key: Optional[any] = None
+    hmac_key: Optional[Union[LazyLoadedArgument,str]] = None
 
     def initialize(self) -> ObjectStore:
         store = ObjectStore.from_file_arguments(self.storage_type, **self.arguments)
```

Comment:
Here we can look to what `Target` does. [Here](https://github.com/nodestream-proj/nodestream/blob/acc6c9500d6cc036bbdbea2d0827bbbaaf77f8b1/nodestream/project/target.py#L32), for example. Target leverages the `LazyLoadedArgument#resolve_if_needed` method to shake out the `LazyLoadedArgument` argument instances and give you _raw_ values. Doing so is how we can avoid transporting the file handling logic all the way to the credential utils command. 

```suggestion
        args = LazyLoadedArgument.resolve_if_needed(self.arguments)
        store = ObjectStore.from_file_arguments(self.storage_type, **args)
```